File: output/xCodeEval_remove_8000/C++/Python/5df97ec9e9c8a03dba8e2ddd54d00e50.py
import math

def is_prime(x):
    if x < 2:
        return False
    for i in range(2, int(math.sqrt(x)) + 1):
        if x % i == 0:
            return False
    return True

def f(n):
    if n <= 1:
        return 0
    if n <= 4:
        return 1
    sum = 1
    bl = n // 100000
    for i in range(bl):
        sum += block[i]

    for i in range(max(5, bl * 100000 + 1), n + 1, 4):
        if is_prime(i):
            sum += 1
    return sum

block = [

File: output/GPT3.5-Turbo/xCodeEval_4000/C++/C/5df97ec9e9c8a03dba8e2ddd54d00e50.c
#include <stdio.h>

typedef long long LL;

int is_prime(int x) {
  for (LL i = 3; i * i <= x; i+=2) {
    if (x % i == 0) {
      return 0;
    }
  }
  return 1;
}

int f(int n) {
  if (n <= 1) {
    return 0;
  }
  if (n <= 4) {
    return 1;
  }
  int sum = 1;
  int bl = n / 100000;
  int block[] = {
